assets/js/modules/analytics/components/common/AccountCreate/index.test.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * AccountCreate component tests.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+	createTestRegistry,
+	fireEvent,
+	provideModules,
+	provideSiteInfo,
+	provideUserAuthentication,
+	provideUserInfo,
+	render,
+} from '../../../../../../../tests/js/test-utils';
+import { mockLocation } from '../../../../../../../tests/js/mock-browser-utils';
+import { mockUseInstanceID } from '../../../../../../../tests/js/mock-use-instance-id';
+
+import {
+	EDIT_SCOPE,
+	MODULES_ANALYTICS,
+	PROVISIONING_SCOPE,
+} from '../../../datastore/constants';
+import {
+	GTM_SCOPE,
+	MODULES_ANALYTICS_4,
+} from '../../../../analytics-4/datastore/constants';
+import { createCacheKey } from '../../../../../googlesitekit/api';
+import { getKeys, setItem } from '../../../../../googlesitekit/api/cache';
+import AccountCreate from '.';
+
+describe( 'AccountCreate', () => {
+	mockLocation();
+	// Provide our own version of the `useInstanceId()` hook to avoid instability in test snapshots.
+	mockUseInstanceID();
```

Comment:
This ability to mock `useInstanceId()` has been introduced to avoid the test snapshot failing in CI while passing locally. See [this](https://github.com/google/site-kit-wp/actions/runs/5657568329/job/15326913151?pr=7358) test run for an example of the failures in CI.